tests/func/test_get.py

```diff
@@ -179,3 +180,15 @@ def test_get_from_non_dvc_master(tmp_dir, erepo_dir, caplog):
 
     assert caplog.text == ""
     assert (tmp_dir / dst).read_text() == "some_contents"
+
+
+def test_get_url(tmp_dir, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen("foo", "foo")
+        assert main(["get", ".", "foo", "--show-url"]) == 0
+
+
+def test_get_url_not_existing(tmp_dir, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen("foo", "foo")
+        assert main(["get", ".", "food", "--show-url"]) == 1
```

Comment:
to pinpoint that its invalid command, you can go with `non-existing-file` instead of `food`

Reply:
Also, as mentioned above, lets make this assert inside tmp_dir. That way we don't have to use `erepo_dir.chdir()` at all.